src/lib/jdk/JdkHelper.ts

```diff
@@ -58,23 +65,21 @@ class JdkHelper {
 
   /**
    * Returns information from the Java executable location, based on the config and platform.
-   * @returns {String} the value where the Java executables can be found
+   * @returns {string} the value where the Java executables can be found
    */
-  getJavaBin() {
+  getJavaBin(): string {
     return this.joinPath(this.getJavaHome(), 'bin/');
   }
 
   /**
    * Returns a copy of process.env, customized with the correct JAVA_HOME and PATH.
-   * @returns {Object} an env object configure to run JDK commands
+   * @returns {NodeJS.ProcessEnv} an env object configure to run JDK commands
    */
-  getEnv() {
-    const env = Object.assign({}, this.process.env);
+  getEnv(): NodeJS.ProcessEnv {
+    const env: NodeJS.ProcessEnv = Object.assign({}, this.process.env);
     env['JAVA_HOME'] = this.getJavaHome();
     env[this.pathEnvironmentKey] =
         this.getJavaBin() + this.pathSeparator + env[this.pathEnvironmentKey];
```

Comment:
Why are you constructing the path manually here instead of using `joinPath`?

Reply:
This is actually editing the PATH environment variable. (eg: PATH="/jdk/bin:/example/bin"), not a file path. I'll add a comment explaining that.

Reply:
Added the comment